test/test_script.py

```diff
@@ -324,4 +324,11 @@ def test_cmd_with_not_arguments(self):
             self.parser.eval("$noargstest()")
         except ScriptError:
             self.fail("Function noargs raised ScriptError unexpectedly.")
-        
+
+    def test_cmd_unset(self):
```

Comment:
There was no test for `unset()` ? :(
Can you please test the simple cases too (including the _/~ case) ?